README.md

```diff
@@ -47,6 +49,11 @@ Specific data download/preprocessing instructions are available at: https://gith
 This repository downloads labeled single-cell data from a specific version of the [mitocheck_data](https://github.com/WayScience/mitocheck_data) repository.
 For more information see [0.download_data/](0.download_data/).
 
+### Supplementary Table 1 - Full list of JUMP-CP phenotype enrichment
+
+We report the top 100 most enriched treatments per phenotype in Supplementary Table 1 of our paper.
+See [`jump_compare_cell_types_and_time_across_phenotypes.tsv.gz`](https://github.com/WayScience/phenotypic_profiling_model/blob/6cd37b2e9255892bed703c56d466806ea63d7066/3.evaluate_model/jump_phenotype_profiles/jump_compare_cell_types_and_time_across_phenotypes.tsv.gz) for the full list.
```

Comment:
Just a small idea, maybe it would be cool to include an example of the TSV in the README here with the top 5 enriched treatments. It would give a little sneak peek! But totally not needed since someone can just download the compressed file to see lol.

Reply:
I like this idea, but will skip to save time. I believe that the most common entry point to this file will be folks who read the paper and then get redirected here after viewing the truncated Sup Table 1